domain-models-interface-extensions/src/test/java/org/folio/rest/tools/PomReaderTest.java

```diff
@@ -0,0 +1,57 @@
+package org.folio.rest.tools;
+
+import io.vertx.core.logging.LoggerFactory;
+import java.io.IOException;
+import java.util.List;
+import org.apache.maven.model.Dependency;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.anyOf;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class PomReaderTest {
+  static {
+    System.setProperty(LoggerFactory.LOGGER_DELEGATE_FACTORY_CLASS_NAME, "io.vertx.core.logging.Log4j2LogDelegateFactory");
+  }
+
+  @Test
+  void testGetModuleName() {
+    assertThat(PomReader.INSTANCE.getModuleName(), is("raml_module_builder"));
+  }
```

Comment:
Can you add a unit test *Test.java (reading from file system) and an integration test *IT.java (reading from jar) for domain-models-runtime-it that checks that getModuleName() returns domain-models-runtime-it?